src/asm/fstack.c

```diff
@@ -337,7 +382,10 @@ void fstk_RunInclude(char const *path)
 	contextStack->lexerState = lexer_OpenFile(fileInfo->name);
 	if (!contextStack->lexerState)
 		fatalerror("Failed to set up lexer for file include\n");
-	lexer_SetStateAtEOL(contextStack->lexerState);
+	if (fromArgv)
+		lexer_SetState(contextStack->lexerState);
+	else
+		lexer_SetStateAtEOL(contextStack->lexerState);
```

Comment:
@ISSOtm Is this too obfuscated?
```suggestion
	(fromArgv ? lexer_SetState : lexer_SetStateAtEOL)(contextStack->lexerState);
```

Reply:
The current way is better.